### PR TITLE
Update es-ES.properties

### PR DESCRIPTION
**Pull request by elKaZe**
Updated, completed, and corrected ;)

### CODE REVIEW

File: locale/es-ES.properties [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
-warningRequiredPrefs = You have not entered all required preferences.
-warningBitlyUnauthorized = Unauthorized. \nYou have to run the Bitly OAuth flow.
+warningRequiredPrefs = No completaste todas las preferencias necesarias.
+warningBitlyUnauthorized = No autorizado. \nDebés completar los pasos del OAuth de Bitly.
 errorTitle = Easyshare: ERROR
-errorNoToken = Unable to get the authenticity token. \nTry to check the pod URL.
-errorLoginFailed = Invalid username or password.
-errorUnknownAspect = Aspect name you entered is unknown.
-errorXhrStatusError = Oops! The server returned an error (%d).
+errorNoToken = errorNoToken = Imposible obtener el token de autentificación. \nRevisá la URL del Pod.
```
Comment (arlogn) [inline locale/es-ES.properties]: The identifier is duplicated

**Review by arlogn — CHANGES_REQUESTED** [review]
Good, thanks. I've left a comment about a little error that should be fixed before to merge the pull request.

**Comment by elKaZe** [comment]
done, sorry I've pasted this text en libreoffice to check the accents, I think that i made a mistake

**Review by arlogn — APPROVED** [review]

**Comment by arlogn** [comment]
Thank you.

**Pull request merged**